roofit/multiprocess/src/Messenger.cxx

```diff
@@ -63,9 +62,9 @@ Messenger::Messenger(const ProcessManager &process_manager)
 {
    sigemptyset(&ppoll_sigmask);
 
-   char addr_prefix_template[30] = "ipc:///tmp/%d_roofitMP";
-   char addr_prefix[50];
-   char addr[100];
+   auto makeAddrPrefix = [](pid_t pid) -> std::string {
+      return "ipc:///tmp/" + std::to_string(pid) + "_roofitMP";
```

Comment:
should not you use RooFit::tmpPath() here ?

Reply:
Thanks for noticing, yes it should be used here! The change got lost when I factored my developments into two commits, thanks for spotting!